src/asRendition.tsx

```diff
@@ -100,6 +101,25 @@ export const withTooltip = (Base: React.ComponentType) => {
 	});
 };
 
+export const withConfirmation = (Base: React.ComponentType) => {
```

Comment:
I don't think we need this, since we only want the confirmation to be available to the `button` component for a start. We can always add it in the future if necessary, but let's start with a more limited usage